src/Libraries/PythonNodeModels/Properties/Resources.resx

```diff
@@ -176,4 +176,7 @@
   <data name="PythonNodeContextMenuEngineVersionTwo" xml:space="preserve">
     <value>IronPython2</value>
   </data>
+  <data name="PythonScriptEditorMoreInfoButton" xml:space="preserve">
+    <value>More Information</value>
+  </data>
```

Comment:
Given this is temporary, maybe we should keep `Save Changes` as is and make this `More info` instead.

Also, I find it worrying that we need to change texts to make up for space. In other languages these texts could be longer and I hope that is not breaking thee layout. Do we need to make additional changes to provide the flexibility?

Reply:
This worries me as well. I changed this because for other buttons we are using `Revert` `Run`, etc so I would like to be consistent, if we can move to use icons with tooltips, that would be even better IMO. This python editor did not get any UX love for so many years.. I would love to get our UX to give it some review :)

`More info` is fine, I did this version mainly wanted to align with the migration dialog, they are doing pretty much the samething becaue